src/uploads.py

```diff
@@ -41,10 +47,10 @@ def dir_listing(req_path):
 
 @app.route("/uploads", methods=["POST"])
 def uploads():
-
     headers = request.headers
     auth = headers.get("X-Api-Key")
-    if not auth in app.config["API_KEYS"]:
+    if not check_valid_key(auth):
```

Comment:
Provide a log message so that log monitoring software can be used to track excessive API key failures.